src/lib/web-worker/worker-iframe.ts

```diff
@@ -35,10 +35,18 @@ export const patchHTMLIFrameElement = (WorkerHTMLIFrameElement: any, env: WebWor
 
     src: {
       get() {
-        let src = getIframeEnv(this).$location$.href;
+        let src = getter(this, ['src']);
```

Comment:
Is there a way to do this without using `getter()` which is doing a call to the main thread? Could you try using `getInstanceStateValue()` and `setInstanceStateValue()` value instead so the value stays on the worker thread? The anchor element does something like this: https://github.com/BuilderIO/partytown/blob/1e2fac14760895e8e6d8624236ae550b584b2995/src/lib/web-worker/worker-anchor.ts#L14

Reply:
Thanks for that 🙏 I just used `getInstanceStateValue()` and `setInstanceStateValue()` to avoid calls to the main thread